src/borg/helpers.py

```diff
@@ -269,6 +269,17 @@ def get_keys_dir():
     return keys_dir
 
 
+def get_key_nonces_dir():
+    """Determine where to store the local nonce high watermark"""
+
+    xdg_config = os.environ.get('XDG_CONFIG_HOME', os.path.join(get_home_dir(), '.config'))
+    nonces_dir = os.environ.get('BORG_KEY_NONCES_DIR', os.path.join(xdg_config, 'borg', 'key-nonces'))
```

Comment:
My thinking was it’s where the nonces for the keyfiles/repokeys are stored. Also key does have the right associations for people who do not even know what a nonce is.